src/ostree/ot-builtin-fsck.c

```diff
@@ -94,7 +96,12 @@ fsck_one_object (OstreeRepo            *repo,
               (void) ostree_repo_delete_object (repo, objtype, checksum, cancellable, NULL);
               object_missing = TRUE;
             }
-          else
+          else if (opt_all)
+            {
+              *out_found_corruption = TRUE;
+              g_printerr ("%s\n", temp_error->message);
+            }
+          else if (!opt_all)
```

Comment:
Minor: let's just use `else` here? It's clearly equivalent, but it parses better. :)